dali/operators/reader/loader/file_loader.h

```diff
@@ -38,7 +38,23 @@ struct ImageFileWrapper {
   Tensor<CPUBackend> image;
   std::string filename;
   // some field for auxiliary info to pass to the reader
-  std::string meta;
+  bool fortran_order;
+
+  const TypeInfo& type() const {
+    return image.type();
+  }
+
+  void set_type(const TypeInfo& type_info) {
+    image.set_type(type_info);
+  }
+
+  const TensorShape<>& shape() const {
+    return image.shape();
+  }
+
+  void set_shape(const TensorShape<>& shape) {
+    image.Resize(shape);
+  }
```

Comment:
Why? This seems irrelevant for plain FileReader.

Reply:
We have code that needs to run for both ImageFileWrapperGPU and ImageFileWrapper. In ImageFileWrapper we use `entry.image.shape` while in ImageFileWrapperGPU we use `entry.shape` (the "image" field is not used there). The code here is offer a uniform API for those two structs.